samples/csharp_dotnetcore/14.nlp-with-dispatch/NlpDispatch/NlpDispatchBot.cs

```diff
@@ -27,17 +27,17 @@ public class NlpDispatchBot : IBot
         /// <summary>
         /// Key in the Bot config (.bot file) for the Home Automation Luis instance.
         /// </summary>
-        public static readonly string HomeAutomationLuisKey = "Home Automation";
+        public static readonly string HomeAutomationLuisKey = "homeautomation.luis";
```

Comment:
For some reason, my bot file ended up with a name of "Home Automation" for this service.